solvers/relay_constellation/umcf_srr_contact_plan/src/srr.py

```
@@ -0,0 +1,399 @@
+            if p is not None:
+                unique_paths[p.nodes] = p
+            continue
+        # Expand neighbors in deterministic order for reproducibility
+        for neighbor, _ in sorted(adjacency.get(node, []), key=lambda x: x[0]):
+            if neighbor in visited:
+                continue
+            # No intermediate ground transit
+            if neighbor in endpoint_ids and neighbor != destination:
+                continue
+            stack.append(
+                (neighbor, path_nodes + [neighbor], visited | {neighbor})
```

Ruff (0.15.11)

[warning] Consider `[*path_nodes, neighbor]` instead of concatenation

Replace with `[*path_nodes, neighbor]`

(RUF005)